recipes-wpe/wpe/wpe_0.1.bb

```diff
@@ -186,4 +186,7 @@ RDEPS_MEDIASOURCE += "${RDEPS_EXTRA}"
 RDEPS_VIDEO += "${RDEPS_EXTRA}"
 RDEPS_WEBAUDIO += "${RDEPS_EXTRA}"
 
-RRECOMMENDS_${PN} += "ca-certificates"
+RRECOMMENDS_${PN} += " \
+    ca-certificates \
+    ttf-bitstream-vera \
```

Comment:
usually fonts have commercial licensing aspects and fact that there are sample WPE images, I would think its not going to be bad out of box experience since I expect such users to build provided images and for folks who want
to then customize can add it. 

We already have BAD_RECOMMENDATIONS and this will be just another one, its not a problem, however if everyone
is overriding it then there is a problem.